client/components/category-pill-navigation/index.tsx

```diff
@@ -1,13 +1,14 @@
 import { Button } from '@wordpress/components';
 import { useEffect, useState, useRef } from '@wordpress/element';
+import { Icon, chevronRight } from '@wordpress/icons';
 import classnames from 'classnames';
 import { LocalizedLink } from 'calypso/my-sites/patterns/components/localized-link';
 
 import './style.scss';
 
 type CategoryPillNavigationProps = {
 	buttons?: {
-		icon: string;
+		icon: JSX.Element;
```

Comment:
```suggestion
		icon: React.ReactElement< typeof Icon >;
```
I guess we could maybe be more specific here if we wanted..?